client/src/FormFrontend/FormFrontend.js

```diff
@@ -412,12 +445,23 @@ class FormFrontend extends React.Component {
                             )
                             .value();
 
+                          const keys_staying_completed = _.chain(template)
+                            .pickBy(
+                              (field_info) => !!field_info["stay_completed"]
+                            )
+                            .keys()
+                            .value();
```

Comment:
Interesting, maybe consider for a follow up issue that "stay completed" forms could use local storage to have memory between sessions. Don't try to do that in this PR though, it's going to require some thought (should storage key off a hash of the field's content so it busts as needed? should forms include a semantic versioning field that could, along with being generally useful for logging, be used for the local storage busting?)